process/HomeworkForMeetings.md

```diff
@@ -0,0 +1,24 @@
+# Homework for meetings
+
+Discussions we have in the WebGPU meetings usually require a lot of context to be productive.
+This is both because we need to know how all 3 native APIs work and because we try to find the right tradeoffs given many constraints.
+Having members of the group produce investigations and proposals is the best way to share context to the group, but it needs to arrive in a timely manner to be useful.
+It happened that multi-page investigations were posted 5 minutes before the start of the meeting in which they were to be discussed.
+This obviously makes it impossible for other members to catch up and explaining the investigation step-by-step in the meeting is a waste of people's valuable time.
+The reverse situation happened too were investigations were posted days (or weeks) in advance but members didn't take/have the time to read them in time, resulting in the same waste of time.
+
+This proposal tries to codify what the expectations are on the members to make meetings happen smoothly.
+
+## Producer homework
+
+In order for members to have time to review material for a meeting, the meat of investigations and proposals have to be posted at least 24 working day hours in advance.
```

Comment:
I would prefer that review material be posted 2 days in advance instead of 1 day in advance. That gives reviewers the opportunity to respond to the pull request in Github, thus giving the producer time to ponder the feedback.  With only one day of notice, reviewers are unlikely going to take the time to respond in Github thus leaving the producer to hear feedback for the first time in the meeting.  

I do not feel strongly about this but if lively discussion is happening on Github, it might be better to keep it there and only discuss in the meeting when the Github discussion has either died or an agreement cannot be made.

Reply:
I think we can use discussions here to express the positions and argue, eventually reaching consensus like we did for #103. However during the call we can "stamp" the consensus and make it "official", thus merging the PR and figuring out the next steps.

Reply:
Done (24 -> 48)